packages/matter.js/src/device/PairedNode.ts

```diff
@@ -374,7 +374,12 @@ export class PairedNode {
             updateTimeoutHandler: async () => {
                 logger.info(`Node ${this.nodeId}: Subscription update not received ...`);
                 try {
-                    await this.subscribeAllAttributesAndEvents();
+                    const { attributeChangedCallback, eventTriggeredCallback } = this.options;
+                    await this.subscribeAllAttributesAndEvents({
+                        ignoreInitialTriggers: true,
```

Comment:
That's a good question depending on how long it stayed disconnected we could miss data. So better miss them all as change callbacks or get change-calls that are for the same value. Maybe second is better or? Better more then missing? Then this should be false.